src/lib/decodeData.ts

```diff
@@ -123,6 +148,7 @@ export const decodeTupleKeyValue = (
     const regexMatch = valueTypePart.match(tupleValueTypesRegex);
 
     if (!regexMatch) {
+      if (valueTypePart === 'address') bytesLengths.push(20);
       return;
```

Comment:
Shouldn't a `else` be added here? I mean the additionnal `if` is useless if after it, it return;